app/Http/Controllers/ServiceTemplateController.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\DeviceGroup;
+use App\Models\Service;
+use App\Models\ServiceTemplate;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+use LibreNMS\Services;
+use Toastr;
+
+class ServiceTemplateController extends Controller
+{
+    public function __construct()
+    {
+        $this->authorizeResource(ServiceTemplate::class, 'service_template');
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response|\Illuminate\View\View
+     */
+    public function index()
+    {
+        return view('service-template.index', [
+            'service_templates' => ServiceTemplate::orderBy('name')->get(),
+            'device_groups' => DeviceGroup::orderBy('name')->get(),
```

Comment:
Add a relationship in the models between ServiceTemplate and DeviceGroup, then change to;
```suggestion
            'device_groups' => DeviceGroup::with('serviceTemplates')->orderBy('name')->get(),
```